### ISSUE TITLE
Enable Heading level selector on WYSIWYG editor toolbar

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9517]
## Issue Description
Editors can add text for description, author guidelines and instructions to users. However, if they need to organize a long text using headings (h2, h3, h4 ...) they will have to open the source code option and edit the HTML directly. 
Currently, the editor toolbar for some textboxes shows buttons for: bold, italic, link, bullet list, insert images, and edit HTML code. Including heading levels would help them to format their texts. 

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question
1. Log in to a journal
2. Click Journal > Masthead > scroll down to Description
3. See the toolbar missing Heading selector

## What application are you using?
 OJS, OMP or OPS version 3.4

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Chrome
- Version: 120

## Testing method
- Method:  Manual; Keyboard

## Action
It is recommended that this toolbar shows only Heading level 2 and below since the heading level 1 (H1) is usually the page title where the text will be inserted.

## Additional information
Notice the buttons to include and edit HTML code.
![image](https://github.com/pkp/pkp-lib/assets/4992902/2caadd1b-5de2-4ed1-97a8-41e293dcbe99)
